packages/pilot/src/containers/TransactionsList/Charts.js

```diff
@@ -48,27 +50,27 @@ const renderLegend = (legendKey) => {
 
 const formatCurrency = value => currencyFormatter(value)
 
-const Charts = ({ data, legendsTitle }) => (
+const Charts = ({ data, legendsTitle, t }) => (
   <Fragment>
     <ResponsiveContainer width="100%" height={380}>
       <BarChart
         data={data}
         height={380}
         maxBarSize={17}
```

Comment:
Quando os valores da transação tem 3 digitos + "mil" o valor dos ticks sobrepõe a legenda do gráfico.

![image](https://user-images.githubusercontent.com/46823713/66855558-dcf9d880-ef59-11e9-8800-b658b79cdca7.png)

Se adicionar um `margin left: 15` no barChart isso deixa de ocorrer.

```suggestion
        maxBarSize={17}
        margin={{
         left: 15,
        }}
```

Reply:
Resolvido!